config.py

```diff
@@ -13,6 +13,7 @@ class Config:
     SESSION_COOKIE_PATH = '/'
     SESSION_COOKIE_HTTPONLY = True
     SESSION_COOKIE_SECURE = True
+    SESSION_COOKIE_SAMESITE = 'Lax'
```

Comment:
This is a new config value in Flask v1.0 and further mitigates against CSRF:

http://flask.pocoo.org/docs/1.0/security/#security-cookie
https://www.owasp.org/index.php/SameSite